platform-coop-toolkit/pcc-framework
# Add WordPress 5.3 support (resolves #77)

## Description

This PR:

- Handles deprecation of the `wp.editor.ServerSideRender` component
- Adds experimental support for [`wp-scripts env`](https://developer.wordpress.org/block-editor/packages/packages-scripts/#env)

## Steps to test

1. Upgrade your development environment to WordPress 5.3-beta3.
2. Create a new post or page and add all of this plugin's custom blocks.

**Expected behavior:** No deprecation errors appear in the browser console.

## Additional information

Not applicable.

## Related issues

- Resolves #77.
